pkgs/development/python-modules/qreactor/default.nix

```diff
@@ -24,14 +23,13 @@ buildPythonPackage rec {
     twisted qtpy
   ];
 
-  checkInputs = [
-    pyqt5
-  ];
-
   pythonImportsCheck = [
     "qreactor"
   ];
 
+  # no tests executed
+  doCheck = false;
```

Comment:
Why remove this? While there are no explicit tests, it does test compiling the code works, right?

Reply:
> Why remove this?

I greped for ``Ran 0 tests in 0.000s`` in the logs and disabled tests in all matching files.

> While there are no explicit tests, it does test compiling the code works, right?

There is a pythonImportsCheck which is enough.

Reply:
> There is a pythonImportsCheck which is enough.

If it's equivalent, why does `pythonImportsCheck` not need the pyqt5 input?

Reply:
Also I'm still not sure what the advantage of explicitly disabling checks is: it just means when upstream ever starts adding tests we're likely to miss them.

Anyway, even if I don't understand the motivation, I'm not opposed to the change. Perhaps at least say something about it in the commit message?

Reply:
> If it's equivalent, why does pythonImportsCheck not need the pyqt5 input?

It takes propagatedBuildInputs and does something like ``python -c 'import X'``

> Also I'm still not sure what the advantage of explicitly disabling checks is

We do not surprisingly execute tests when updating.

> it just means when upstream ever starts adding tests we're likely to miss them.

That is the duty of the maintainer to check. For Python the importsCheck is usually enough and catches the same bugs as running the tests itself for small packages. Most other test failures are logic mistakes.

> Perhaps at least say something about it in the commit message?

This is the 5th PR of this kind and to be honest I am to lazy to do a proper description.

Reply:
> We do not surprisingly execute tests when updating.

Surprisingly *not* executing tests seems worse to me...